compiler/one-cmds/tests/onecc_045.test

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="./reshape_matmul.circle"
+configfile="onecc_045.cfg"
+outputfile="reshape_matmul.onecc_045.q.circle"
+
+rm -f ${filename}.log
+rm -f ${filename}.first.cdump
+rm -f ${filename}.second.cdump
+rm -f ${outputfile}
+
+if [[ ! -s "${inputfile}" ]]; then
+  one-import onnx -i reshape_matmul.onnx -o ${inputfile}
+fi
```

Comment:
I think it would be better to prepare this in `prepare_test_materials.sh`
Please add a TODO to create circle file in `prepare_test_materials.sh`
Found out that this circle generated in `one-import-onnx_002.test`
Thus would be better to rename circle file name in `one-import-onnx_002.test` not to delete this circle file.

Reply:
> I think it would be better to prepare this in prepare_test_materials.sh

I agree with this. I've added the comment. Thanks.